### DISCUSSION TITLE
how to get a badge in github?

### DISCUSSION THREAD

**Discussion by m0vy** [community/community#21229] [Other Feature Feedback, Questions, & Ideas]
how to get a badge in github?

**Comment by radar-blip** [comment]
@m0vy, Here is some info for you on getting badges: 
https://github.com/Schweinepriester/github-profile-achievements#achievements

**Comment by ghost** [comment]
ty

**Comment by Ismoh** [comment]
I love to use [shields.io](https://shields.io/) or badgen, etc pp.
There are default badges, examples and you can create custom static badges!
I hope this helps!

@m0vy dont forget to select a post as an answer!

**Reply by Ismoh** [reply]
I realised @m0vy was talking about githubs profile achievements and not README.md badges, which you create yourself.

**Comment by davidaMeteo** [comment]
you can open a pull request, select a reviewer and then merge it yourself without a review

**Reply by Ismoh** [reply]
I don't understand what you even mean by this.

**Reply by davidaMeteo** [reply]
See here: https://github.com/orgs/community/discussions/38626
